src/ui/linkformview.js

```diff
@@ -77,6 +84,37 @@ export default class LinkFormView extends View {
 		 */
 		this.cancelButtonView = this._createButton( t( 'Cancel' ), cancelIcon, 'ck-button-cancel', 'cancel' );
 
+		/**
+		 * A reference to {@link module:link/linkcommand~LinkCommand#manualDecorators manual decorators}
+		 * collection of the {@link module:link/linkcommand~LinkCommand}.
+		 *
+		 * Helps bootstrap the {@link #manualDecoratorsUIView UI} so it corresponds with the global
+		 * configuration of manual decorators and to synchronize its state later on when the user
+		 * is editing the content.
+		 *
+		 * @readonly
+		 * @type {model:utils/collection~Collection}
+		 */
+		this.manualDecorators = manualDecorators;
+
+		/**
+		 * A collection of {@link module:ui/button/switchbuttonview~SwitchButtonView},
+		 * which corresponds to {@link #manualDecorators manual decorators} configured in the editor.
+		 * Populated by {@link #_createManualDecoratorsUIView}.
+		 *
+		 * @readonly
+		 * @type {module:ui/viewcollection~ViewCollection}
+		 */
+		this.manualDecoratorsUIView = this._createManualDecoratorsUIView();
```

Comment:
It's not a view. It's a view collection. This should be clear in the name of the property.

Also: a public property should not link to private method `_createManualDecoratorsUIView`. I think this property does not need to be public in the first place.

Also: I see that this is a collection of switches so this also must be clear. Something like `manualDecoratorSwitches` would fit better.